### PR TITLE
Updates for 1.7 cadvisor changes and rbac requirements

### PR DESCRIPTION
**Pull request by zatricky**
Notes:
- I'm not enabling rbac - but have at least made appropriate changes to the commented rbac rules. This works with rbac enabled with kubeadm-generated clusters.
- I've included joshes' /var/lib/grafana fix, though I see he also has a separate pull request for that

See also:
- prometheus/prometheus#2606
- prometheus/prometheus#2918
- kubernetes/charts#1655
- kubernetes/kubernetes#48483

### CODE REVIEW

**Comment by MarcelMue** [comment]
This is breaking for K8s 1.7.2 and below, correct? If so, then please add a note about this being the case in the readme.

I will do some testing from my side shortly.

**Comment by zatricky** [comment]
I'll see if I can roll out some older 1.7.x clusters to test.

**Comment by dominikhahn** [comment]
Tested on 1.7.6 cluster. Works fine. Thanks @zatricky !

File: manifests/prometheus/configmap.yaml [hunk 1]
```diff
@@ -104,6 +104,23 @@ data:
...
             regex: 9\d{3}
+      - job_name: 'kubernetes-cadvisor'
+        scheme: https
+        tls_config:
+          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+        kubernetes_sd_configs:
+          - role: node
+        relabel_configs:
+          - action: labelmap
```
Comment (r4j4h) [inline manifests/prometheus/configmap.yaml]: I don't know if it's harmful or not, but I think this extra `action: labelmap` with no other keys is unnecessary.

File: configs/prometheus/prometheus.yaml [hunk 2]
```diff
@@ -101,3 +101,20 @@ scrape_configs:
...
         regex: 9\d{3}
+  - job_name: 'kubernetes-cadvisor'
+    scheme: https
+    tls_config:
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    kubernetes_sd_configs:
+      - role: node
+    relabel_configs:
+      - action: labelmap
```
Comment (r4j4h) [inline configs/prometheus/prometheus.yaml]: [This comment](https://github.com/giantswarm/kubernetes-prometheus/pull/73/files#r142479485) may apply here as well

**Comment by zburgermeiszter** [comment]
Closes:
#79 #90 #89 #88 #86 #84

**Pull request merged**